src/Camera.jl

```diff
@@ -1,12 +1,18 @@
 using JulGame
 using .Math
+import JulGame: deprecated_get_property
 
 mutable struct Camera
     dimensions::Vector2
     offset::Vector2f
     position::Vector2f
     startingCoordinates::Vector2f
-    target#EntityModule.Entity
+
+    target::Union{
+        Ptr{Nothing}, 
+        # JulGame.EntityModule.Entity,
```

Comment:
I originally assumed the target of the camera would be an entity, but it looks like it is always a `Transform`.

Reply:
Yep I definitely messed that comment up 😅

Reply:
Oh! Glad that was you not me! No worries!